src/app/entry.service.ts

```diff
@@ -1,19 +1,22 @@
 import { Injectable } from '@angular/core';
 import {HttpClient} from '@angular/common/http'
+import { Subject } from 'rxjs';
 
 @Injectable({
   providedIn: 'root'
 })
 export class EntryService {
-  url: string = "http://localhost:3000/user/";
```

Comment:
There should be a separate constant /config file that contains all config releated data.